recipes/galaxy-schema/meta.yaml

```
@@ -0,0 +1,81 @@
+{% set version = "24.1.1" %}
+{% set sha256 = "622c65bf1f7fc0cfb5cf6ee844e819a7ceff2b929aca5894848761b3a7df84fa" %}
+{% set galaxy_version = version.split(".")[:2]|join(".") %}
+
+package:
+  name: galaxy-schema
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/g/galaxy-schema/galaxy_schema-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  run_exports:
+    - {{ pin_subpackage('galaxy-schema', max_pin="x") }}
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+  run:
+    - python >=3.8
+    - galaxy-util >={{ galaxy_version }}
+    - pydantic >=2.7.4
+    - email-validator  # pydantic[email] dependency
+    - email_validator  # pydantic[email] dependency
```

Maybe it's the comment that confuses conda-build?

```suggestion
    - email-validator
```

---

Is that why? comments not allowed? 🤯

---

I suppose it can be confused with a [selector](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors)?